types/construction_preprocess_request.go

```diff
@@ -21,18 +21,9 @@ package types
 // it needs to request for construction. Metadata provided in this object should NEVER be a product
 // of live data (i.e. the caller must follow some network-specific data fetching strategy outside of
 // the Construction API to populate required Metadata). If live data is required for construction,
-// it MUST be fetched in the call to `/construction/metadata`. The caller can provide a max fee they
-// are willing to pay for a transaction. This is an array in the case fees must be paid in multiple
-// currencies. The caller can also provide a suggested fee multiplier to indicate that the suggested
-// fee should be scaled. This may be used to set higher fees for urgent transactions or to pay lower
-// fees when there is less urgency. It is assumed that providing a very low multiplier (like 0.0001)
-// will never lead to a transaction being created with a fee less than the minimum network fee (if
-// applicable). In the case that the caller provides both a max fee and a suggested fee multiplier,
-// the max fee will set an upper bound on the suggested fee (regardless of the multiplier provided).
+// it MUST be fetched in the call to `/construction/metadata`.
 type ConstructionPreprocessRequest struct {
-	NetworkIdentifier      *NetworkIdentifier     `json:"network_identifier"`
```

Comment:
why removed it?

Reply:
That was done by `codegen.sh`.  
I did not remove anything.

This PR consists of exactly 3 line changes to *only* the `codegen.sh` file, followed by a run of `bash codegen.sh`.

1. https://github.com/sleepdefic1t/mesh-sdk-go/blame/88d8e632c9852797f90b8058d03f9735ec7acf01/codegen.sh#L56
2. https://github.com/sleepdefic1t/mesh-sdk-go/blame/88d8e632c9852797f90b8058d03f9735ec7acf01/codegen.sh#L128
3. https://github.com/sleepdefic1t/mesh-sdk-go/blame/88d8e632c9852797f90b8058d03f9735ec7acf01/codegen.sh#L136

Reply:
Hi @jingweicb 

Noting that #479 (your team’s fix) is unmerged and out of scope here. Can we mark this resolved?

Reply:
can you add them back manually ?

Reply:
@jingweicb 

Reverted: https://github.com/coinbase/mesh-sdk-go/pull/509/commits/092fb696b40900569fe6be2f4b110cf1ca03dc72